File: src/main/java/org/example/component/column/TypeHTML.java
package org.example.component.column;

import org.example.component.Column;

public class TypeHTML extends Column {
    public TypeHTML(String name) {
        super(name);
        this.type = TypeColumn.HTML.name();
    }

    @Override
    public boolean validate(String value) {
        return value.toLowerCase().endsWith(".html");
    }
}

File: Classes/TypeHTML.cs
using System;
using WebApplication.Classes;

namespace WebApplication.Classes
{
    public class TypeHTML : Column
    {
        public TypeHTML(string name) : base(name)
        {
            Type = TypeColumn.HTML.ToString();
        }

        public override bool Validate(string value)
        {
            return value.ToLower().EndsWith(".html");
        }
    }
}
